app/javascript/mastodon/features/account/components/account_note.js

```diff
@@ -92,6 +101,27 @@ class AccountNote extends ImmutablePureComponent {
     this.textarea = c;
   }
 
+  setEditable () {
+    const sleep = (waitSeconds) => {
+      return new Promise(resolve => {
+        setTimeout(() => {
+          resolve();
+        }, waitSeconds);
+      });
+    };
+    let my = this;
+    sleep(50)
+      .then(() => {
+        my.setState({ editable: true });
+        my.textarea.focus();
+      }).catch(() => {
+        my.setState({ editable: false });
+      });
```

Comment:
Why the delay? (also, `waitSeconds` is a misleading name)

Reply:
To focus the textarea (if no delay, the button(div) will be focused because "clicked")

Reply:
Also, I have to change misleading var name